frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -86,6 +86,25 @@ describe("scenarios > visualizations > pivot tables", () => {
       cy.findByText("3,976");
     });
   });
+
+  it("should be able to use binned numeric dimension as a grouping (metabase#14136)", () => {
+    // Sample dataset Orders > Count by Subtotal: Auto binned
+    cy.visit(
+      "/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJxdWVyeSIsInF1ZXJ5Ijp7InNvdXJjZS10YWJsZSI6MiwiYWdncmVnYXRpb24iOltbImNvdW50Il1dLCJicmVha291dCI6W1siYmlubmluZy1zdHJhdGVneSIsWyJmaWVsZC1pZCIsMTRdLCJkZWZhdWx0Il1dfSwiZGF0YWJhc2UiOjF9LCJkaXNwbGF5IjoiYmFyIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319",
+    );
+    cy.findByText("Count by Subtotal: Auto binned");
+
+    cy.findByText("Visualization").click();
+    cy.get(".Icon-pivot_table").click({ force: true });
+
+    cy.get(".Visualization").within(() => {
+      cy.findByText("Subtotal");
+      cy.findByText("Count");
+      cy.contains(/\d+\s+–\s+\d+/);
```

Comment:
Can this backfire if we ever decide to use different delimiter?

Reply:
Do it with Products table on Rating instead, and assert by `Grand totals` `200`.
![image](https://user-images.githubusercontent.com/1447303/102915120-8f43fe80-4481-11eb-9b8b-5c7a6fde1971.png)

Reply:
Ouch.
I forgot about decimal places.

Anyway, this regex is tied to this specific repro only. Maybe it's redundant and I should remove regex assertion all together? Was there any other reason you asked me to change the query other than to show me that there can be decimal places? @flamber

Reply:
I wanted to avoid too much formatting, and since we just want to assert if visualization works, not really any specific numbers.
Removing the regex, and just asserting `Grand totals` should be fine enough.